storyblok/storyblok-design-system
# fix(on-720): align collapse icon with my account

## Pull request type

Jira Link: [ON-720](https://storyblok.atlassian.net/browse/ON-720)




## How to test this PR
Please check out your DS branch to this PR's branch and test it

- the collapse button doesn't interfere with the scroll bar or the back to v1 link
<img width="932" alt="Captura de ecrã 2022-07-28, às 11 36 24" src="https://user-images.githubusercontent.com/94900845/181485961-8b0eab46-7097-4f3a-9b9a-e71524efd306.png">



## Other information
